docs/courses/level-one/chapter-5/chapter-5.5.md

```diff
@@ -17,66 +19,81 @@ To calculate data and add more functionality to your workflows you can use the C
 ## About the Code node
 
 /// warning | Code node modes
-The Code node has two operational **Modes** that change the way it processes data. The **Run Once for All Items** mode allows you to accumulate data from all items on the input list. The **Run Once for Each Item** is used to add custom snippets of JavaScript code that should be executed once for every item that it receives as the input. Learn more about how to use the [Code node](/integrations/builtin/core-nodes/n8n-nodes-base.code/index.md).
+The Code node has two operational **modes**, depending on how you want to process items:
+
+* **Run Once for All Items** allows you to write code to process all input items at once, as a group.
+* **Run Once for Each Item** executes your code once for each input item.
+
+Learn more about how to use the [Code node](/integrations/builtin/core-nodes/n8n-nodes-base.code/index.md).
 ///
-- Create your own node.
-- Write custom expressions.
-- Use the Code node.
-- Get the most out of n8n.
 
-In n8n, the data that's passed between nodes is an array of objects with the following structure:
+In n8n, the data that's passed between nodes is an array of objects with the following JSON structure:
 
 ```json
```

Comment:
Do you think this new representation works? I found the previous one hard to read.